components/loadercontainer/appearance.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Appearance
+page_title: LoaderContainer Appearance
+description: Appearance settings of the LoadingContainer for Blazor.
+slug: loadercontainer-appearance
+tags: telerik,blazor,loader,container,loadercontainer,appearance
+published: True
+position: 5
+---
+
+# Appearance Settings
+
+The LoaderContainer component provides the following parameters that control its appearance:
+
+* [LoaderContainer Specific](#loadercontainer-specific)
+    * [OverlayThemeColor](#overlaythemecolor)
+* [Shared with the Loader Indicator](#shared-with-the-loader-indicator)
+    * [Type](#type)
+    * [Size](#size)
+    * [ThemeColor](#themecolor)
+    
+You can see the appearance settings in action in the [LoaderContainer Customization](https://demos.telerik.com/blazor-ui/loadercontainer/customization) Live Demo.
+
+## LoaderContainer Specific
+
+### OverlayThemeColor
+
+The `OverlayThemeColor` parameter controls the color of the overlay for the LoaderContainer. It takes a string from the options below. If you provide a `String.Empty`, `null` or invalid option (not listed below) the color of the overlay will be `transparent`.
+
+* `dark` - the default background color.
+* `light`
+
+>caption Change the OverlayThemeColor
```

Comment:
Mash two small screenshots together to show the difference between Light and Dark modes, add a caption to the image to explain what it shows.

Reply:
done